doc/nrf/vulnerabilities.rst

```diff
@@ -8,8 +8,6 @@ Vulnerabilities
    :depth: 2
 
 Here are some known vulnerabilities.
-
-Table
-*****
+You can filter based on the affected versions using the dropdown menu below.
```

Comment:
suggestion for commit sequence: I think it would be better to first add the extension (split if you want) and then relevant documentation pages.